src/EditorFeatures/Core/CommandHandlers/QuickInfoCommandHandlerAndSourceProvider.cs

```diff
@@ -35,19 +35,25 @@ internal partial class QuickInfoCommandHandlerAndSourceProvider :
 
         [ImportingConstructor]
         public QuickInfoCommandHandlerAndSourceProvider(
+            IThreadingContext threadingContext,
```

Comment:
Missing `[Obsolete]` attribute?

Reply:
⌛️ Fixed locally and will be in next commit.